src/main/kotlin/edu/agh/susgame/dto/ClientSocketMessage.kt

```diff
@@ -0,0 +1,38 @@
+package edu.agh.susgame.dto
+
+import edu.agh.susgame.dto.common.GameStatus
+import kotlinx.serialization.Serializable
+
+/**
+ * Message sent by client via WebSocket
+ */
+@Serializable
+sealed class ClientSocketMessage {
+    @Serializable
+    data class HostDTO(
```

Comment:
@Uiskam do czego to będzie wykorzystywane. Zaproponuj proszę jakiś komentarz, żeby tutaj dodać